Sources/Node/Convertibles/Integer+Convertible.swift

```diff
@@ -5,16 +5,16 @@ extension Int32: NodeConvertible {}
 extension Int64: NodeConvertible {}
 
 extension SignedInteger {
-    public func makeNode(in context: Context? = nil) -> Node {
-        let number = Schema.Number(self.toIntMax())
-        return .number(number)
-    }
-
     public init(node: Node) throws {
         guard let int = node.int else {
-            throw NodeError(node: node, expectation: "\(Self.self)")
+            throw Error.unableToConvert(input: node, expectation: "\(Self.self)", path: [])
```

Comment:
`NodeError` is better. It's difficult when working with Vapor to disambiguate error types--especially when there is a type name that is the same as the module name.